### PR TITLE
 Get mcg core and operator version info

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar <shmohan@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/pytest_customization/ocscilib.py [hunk 1]
```diff
@@ -166,6 +162,15 @@ def pytest_configure(config):
...
                 )
+            mods = get_version_info(
+                namespace=ocsci_config.ENV_DATA['cluster_namespace']
+            )
+            skip_list = ['ocs-operator']
+            for key, val in mods.items():
+                if key in skip_list:
+                    pass
+                else:
+                    config._metadata[key] = val.rsplit('/')[-1]
```
Comment (gklein) [inline ocs_ci/framework/pytest_customization/ocscilib.py]: Can be shorter with a negative condition without the else clause:
 
if not key in skip_list:
    config._metadata[key] = val.rsplit('/')[-1]

**Comment by shylesh** [comment]
New output in the report


BUILD_ID | 3010
-- | --
BUILD_NUMBER | 3010
BUILD_TAG | jenkins-qe-deploy-ocs-cluster-3010
BUILD_URL | https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3010/
Ceph Version | 14.2.4-69.el8cp (8d72f97ca776c758a7ce0009959ca3044cd0b9c2) nautilus (stable)
Cluster Version | 4.2.0-0.nightly-2019-12-11-171302
EXECUTOR_NUMBER | 2
GIT_BRANCH | origin/master
GIT_COMMIT | c62d431d2bb8f357130a761e95647a57814f5ebe
GIT_URL | ${JOBS_REPOSITORY}
JENKINS_URL | https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/
JOB_NAME | qe-deploy-ocs-cluster
NODE_NAME | temp-slave-shmohan-versioninfo
OCS operator | 4.2-rc2
Test Run Name | OCS_Downstream_shmohan-AWS-IPI
WORKSPACE | /home/jenkins/workspace/qe-deploy-ocs-cluster
ceph | rhceph:4-120.20191203.ci.0
cephfsplugin | 4.2-226.72ac53b6.release_4.2
noobaa-operator | mcg-operator:2.0.9-36.db6a79b.2.0
noobaa_cor | mcg-core:5.2.11-20.59b35a4f0.5.2
noobaa_db | mongodb-36-rhel7:latest
rbdplugin | 4.2-226.72ac53b6.release_4.2
rook-ceph-operator | rook-ceph:4.2-259.96d5c35a.ocs_4.2
rook_csi_attacher | ose-csi-external-attacher:4.2
rook_csi_ceph | cephcsi:4.2-226.72ac53b6.release_4.2
rook_csi_provisioner | ose-csi-external-provisioner-rhel7:4.2
rook_csi_registrar | ose-csi-driver-registrar:4.2

File: ocs_ci/framework/pytest_customization/ocscilib.py (hunk 1 above)
Comment (shylesh) [reply ocs_ci/framework/pytest_customization/ocscilib.py]: Done

File: ocs_ci/ocs/resources/ocs.py [hunk 2]
```diff
@@ -152,6 +152,18 @@ def delete_temp_yaml_file(self):
...
 
+def get_version_info(namespace=None):
+    package_manifest = PackageManifest(resource_name=defaults.OCS_OPERATOR_NAME)
+    csv_name = package_manifest.get_current_csv()
+    csv_pre = CSV(
+        resource_name=csv_name,
+        namespace=namespace
+    )
+    info = get_images(csv_pre.get())
+    logging.info(info)
```
Comment (petr-balogh) [inline ocs_ci/ocs/resources/ocs.py]: logger.info

**Pull request merged**